drools-ansible-rulebook-integration-api/src/test/java/org/drools/ansible/rulebook/integration/api/AdditionTest.java

```diff
@@ -70,7 +71,7 @@ public void testExecuteRules() {
     public void testExecuteRulesSet() {
         RulesSet rulesSet = new RulesSet();
         rulesSet.addRule().withCondition().all()
-                .addSingleCondition(prototypeField("nested.i"), Index.ConstraintType.EQUAL, prototypeField("nested.j").add(fixedValue(1)));
```

Comment:
Just a note: when we will move the extractor module into Drools I think we could also deprecate (or completely remove) the old `prototypeField`.

Reply:
one step at a time 🙃 
the correct place to mark this would be another ticket 😉